.browserslistrc

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2022 2020-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
I think the text here should be `2020-present` - I don't think the additional `2022` is necessary